lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.html

```diff
@@ -4,8 +4,14 @@
             class="adf-option-chips"
             *ngFor="let option of selectedOptions"
             (removed)="remove(option)">
-            <span>{{option}}</span>
-            <button matChipRemove class="adf-option-chips-delete-button" [attr.aria-label]="('SEARCH.FILTER.BUTTONS.REMOVE' | translate) + ' ' + option">
+            <span [matTooltip]="'SEARCH.RESULTS.WILL_CONTAIN' | translate:{searchTerm: option.fullPath}"
+                  [matTooltipDisabled]="!option.fullPath" [matTooltipShowDelay]="tooltipShowDelay">
```

Comment:
Yes but after we release this, customers might use this component to define filters on some custom properties as well and we can't predict if they will contain longer values. But on the other hand they can always use the `fullPath` so I think we can leave it as it for now and in case of bugs modify later

Reply:
Added tooltips for autocomplete in case of long values as long values shrink there, for chips we need tooltips only for Categories to show the full path.